ci: error.PermissionDenied on syscall during zig-test wokflow

The zig-test GH workflow fails with an `error.PermissionDenied` on `master` w/o any change on code and container.

```
Run zig build test -Dengine=v8
run test: error: Reflect tests: OK
error: PermissionDenied
/usr/local/zig/lib/std/os/linux/io_uring.zig:59:22: 0x5a963f in init_params (test)
            .PERM => return error.PermissionDenied,
                     ^
/usr/local/zig/lib/std/os/linux/io_uring.zig:27:16: 0x5a9fd5 in init (test)
        return try IO_Uring.init_params(entries, &params);
               ^
/__w/zig-js-runtime/zig-js-runtime/vendor/tigerbeetle-io/io/linux.zig:33:28: 0x5aa58d in init (test)
        return IO{ .ring = try IO_Uring.init(entries, flags) };
                           ^
/__w/zig-js-runtime/zig-js-runtime/src/loop.zig:48:16: 0x5ab62b in init (test)
        io.* = try IO.init(32, 0);
               ^
/__w/zig-js-runtime/zig-js-runtime/src/engine.zig:43:16: 0x5abb0f in loadEnv__anon_9434 (test)
    var loop = try Loop.init(alloc);
               ^
/__w/zig-js-runtime/zig-js-runtime/src/run_tests.zig:101:13: 0x5ba94f in main (test)
        _ = try eng.loadEnv(&proto_arena, null, proto.exec);
            ^
/__w/zig-js-runtime/zig-js-runtime/src/test_runner.zig:23:5: 0x5be3c3 in main (test)
    try tests.main();
    ^
run test: error: the following command exited with code 1 (expected exited with code 0):
/__w/zig-js-runtime/zig-js-runtime/zig-cache/o/850bdfb57db55da751ebfe36542b5dd1/test --listen=- 
Build Summary: 2/4 steps succeeded; 1 failed (disable with --summary none)
test transitive failure
+- run test failure
error: the following build command failed with exit code 1:
/__w/zig-js-runtime/zig-js-runtime/zig-cache/o/fe980a747346076740f2612450003fc0/build /usr/local/zig/zig /__w/zig-js-runtime/zig-js-runtime /__w/zig-js-runtime/zig-js-runtime/zig-cache /github/home/.cache/zig --seed 0x4ef7a48c test -Dengine=v8
Error: Process completed with exit code 1.
```
See https://github.com/lightpanda-io/zig-js-runtime/actions/runs/9562815158/job/26359988436 for details.

The last time the test pass was on #229 https://github.com/lightpanda-io/zig-js-runtime/actions/runs/9208534873/job/25330948570

--- a/.github/workflows/benchmark.yml
+++ b/.github/workflows/benchmark.yml
@@ -45,6 +45,11 @@ jobs:
        username: ${{ github.actor }}
        password: ${{ secrets.GITHUB_TOKEN }}
 
+      # docker blocks io_uring syscalls by default now.
+      # see https://github.com/tigerbeetle/tigerbeetle/pull/1995
+      # see https://github.com/moby/moby/pull/46762
+      options: "--security-opt seccomp=unconfined"
+
     steps:
       - uses: actions/checkout@v3
         with:

--- a/.github/workflows/zig-test.yml
+++ b/.github/workflows/zig-test.yml
@@ -107,6 +107,11 @@ jobs:
        username: ${{ github.actor }}
        password: ${{ secrets.GITHUB_TOKEN }}
 
+      # docker blocks io_uring syscalls by default now.
+      # see https://github.com/tigerbeetle/tigerbeetle/pull/1995
+      # see https://github.com/moby/moby/pull/46762
+      options: "--security-opt seccomp=unconfined"
+
     steps:
       - uses: actions/checkout@v3
         with:

--- a/.github/workflows/zig-test.yml
+++ b/.github/workflows/zig-test.yml
@@ -11,6 +11,7 @@ on:
       - "src/**/*.zig"
       - "src/*.zig"
       - "*.zig"
+      - ".github/**"
   pull_request:
 
     # By default GH trigger on types opened, synchronize and reopened.
@@ -26,6 +27,7 @@ on:
       - "src/**/*.zig"
       - "src/*.zig"
       - "*.zig"
+      - ".github/**"
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 

